pkg/mount/dag_steps.go

```diff
@@ -586,3 +586,10 @@ func (s *State) WaitForSysrootDagStep(g *herd.Graph) error {
 			}
 		}))
 }
+
+// LvmActivation will try to activate lvm volumes/groups on the system.
+func (s *State) LvmActivation(g *herd.Graph) error {
```

Comment:
I see LVM is used in other parts, so for consistency I think this one should be renamed to `LVMActivation`

Reply:
done!